uint16 order
uint16 last_order
string sender_id
string data